server/routerlicious/packages/services-client/src/restWrapper.ts

```diff
@@ -73,6 +74,9 @@ export class RestWrapper {
         if (this.defaultHeaders) {
             options.headers = { ...this.defaultHeaders, ...options.headers };
         }
+        if (options.headers === undefined || options.headers["x-correlation-id"] === undefined) {
```

Comment:
`if (!options.headers?["x-correlation-id"])` would be simpler. At least just check if falsy rather than checking if strictly `undefined`?

Reply:
Ditto

Reply:
Sounds good. Changed in the update.